color-picker-project/src/components/hero/Hero.css

```diff
@@ -89,7 +89,14 @@
   z-index: 0;
 }
 
-@media (max-width: 768px) {
+@media (min-width: 420px) {
```

Comment:
As far as I can tell .content-container does not have a default margin-top outside of the media queries.  I'll test after merging into the working branch, but I have a suspicion that the hero section will not have enough margin-top on devices with width less than 420px (i.e., iPhone SE).

Reply:
Suggest removing the media query for `min-width: 420px` and applying these styles `margin-top: 4rem;
    gap: 24px;` to the .content-container selector (lines 13-19).  

At screen widths less than 420px (for example, iPhone SE, iPhone 12/13 mini), hero content is still hidden under the nav bar due to the lack of margin-top.

<img width="949" alt="image" src="https://github.com/chingu-voyages/v49-tier2-team-18/assets/106836794/127771dc-6e79-44ad-84e2-7e4ddbffaf3f">


The margin-top is working as intended above 420px:

<img width="949" alt="image" src="https://github.com/chingu-voyages/v49-tier2-team-18/assets/106836794/d6b3f51c-bc08-4394-994d-f0741e343725">